tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/util/keycloak/KeycloakAdminClient.java

```diff
@@ -62,6 +62,7 @@ public KeycloakAdminClient(final String url, final String realm, final String us
 	 */
 	public void importRealmConfiguration(InputStream is) throws IOException {
 		PartialImportRepresentation piRep = JsonSerialization.readValue(is, PartialImportRepresentation.class);
+		// TODO - fix, the realm name should be passed in as a parameter
 		keycloak.realm("wildfly-realm").partialImport(piRep);
```

Comment:
@fabiobrz isn't the realm name specified in the JSON file to import?

Reply:
Now that you've mentioned it, I see I have been maybe a bit too quick in here.
There's actually some controversial issue around the Keycloak/RH-SSO admin client (also about the compatible versions), and I am going to check it, thanks for bringing it up.

Reply:
Back with some updates. So the answer is "no", meaning that the Keycloak Admin API contracts require for the realm name, to identify the realm where resources will be created.
Here's an example from an existing test:

```
{
  "roles": {
    "realm" : [{
      "name": "user"
    }]
  },

  "users": [{
    "username" : "user",
    "enabled": true,
    "credentials" : [{
      "type" : "password",
      "value" : "password"
    }],
    "realmRoles": ["user"],
    "clientRoles": {
      "account": ["view-profile", "manage-account"]
    }
  }]
}
```
Anyway, I fixed the method as well, according to the `TODO`. Thanks again for bringing this up.